lib/api/CModelSizeStatsJsonWriter.cc

```diff
@@ -45,6 +45,7 @@ const std::string LOG_TIME{"log_time"};
 const std::string CATEGORIZER_STATS{"categorizer_stats"};
 const std::string PARTITION_FIELD_NAME{"partition_field_name"};
 const std::string PARTITION_FIELD_VALUE{"partition_field_value"};
+const std::string JSON_MEMORY_ALLOCATOR_BYTES("json_memory_allocator_bytes");
```

Comment:
nit: I am wondering if this name is not too specific. Maybe `output_memory_allocator_bytes`?